lib/QueryBuilder.jsx

```diff
@@ -24,7 +24,7 @@ export default class QueryBuilder extends React.Component {
             fields: React.PropTypes.array.isRequired,
             operators: React.PropTypes.array,
             combinators: React.PropTypes.array,
-            getEditor: React.PropTypes.func,
+            controls: React.PropTypes.object,
```

Comment:
We should add more explicit check with React.PropTypes.Shape since we know it should have one or more of the following props: `{field,operator}Selector` or `valueEditor`